### PR TITLE
feat(dashboard): page meta, gtm, and uptime

### PR DESCRIPTION
**Pull request by LetItRock**
### What changed? Why was the change needed?

Favicon, Page meta, GTM, Better Stack Uptime

### Screenshots

https://github.com/user-attachments/assets/52ab88c1-9f22-4ebc-b4b9-f79fc9026ac6

### CODE REVIEW

File: apps/dashboard/index.html [hunk 1]
```diff
@@ -2,9 +2,13 @@
...
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React + TS</title>
+    <title>Novu Cloud Dashboard</title>
+    <meta name="description" content="Novu Cloud Dashboard" />
+    <meta name="theme-color" content="#fff" />
+    <link rel="icon" href="/favicon-gradient.svg" />
+    <link rel="apple-touch-icon" href="/favicon-gradient.svg" />
+    <link rel="manifest" href="/manifest.json" />
```
Comment (LetItRock) [inline apps/dashboard/index.html:11]: favicons and manifest

File: apps/dashboard/index.html [hunk 2]
```diff
@@ -15,8 +19,40 @@
...
+        w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != 'dataLayer' ? '&l=' + l : '';
+        j.async = true;
+        j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, 'script', 'dataLayer', '<%= env.VITE_GTM %>');
+    </script>
+    <% } %>
```
Comment (LetItRock) [inline apps/dashboard/index.html:35]: GTM

File: apps/dashboard/index.html [hunk 3]
```diff
@@ -15,8 +19,40 @@
...
+      ></iframe>
+    </noscript>
+    <% } %> <% if (env.VITE_SELF_HOSTED === 'false' ) { %>
+    <script
+      src="https://uptime.betterstack.com/widgets/announcement.js"
+      data-id="144175"
+      async="async"
+      type="text/javascript"
+    ></script>
+    <% } %>
```
Comment (LetItRock) [inline apps/dashboard/index.html:54]: Uptime

File: apps/dashboard/package.json [hunk 4]
```diff
@@ -106,6 +106,7 @@
     "typescript": "5.6.2",
     "typescript-eslint": "^8.0.1",
     "vite": "^5.4.1",
+    "vite-plugin-ejs": "^1.7.0",
```
Comment (LetItRock) [inline apps/dashboard/package.json:111]: the Vite EJS plugin that allows the use of conditionals in the HTML template; by default, it's not supported by Vite :/

File: apps/dashboard/src/components/workflow-editor/configure-workflow.tsx [hunk 5]
```diff
@@ -16,124 +16,130 @@ import { Switch } from '../primitives/switch';
...
-            <FormItem>
-              <FormLabel>Workflow Identifier</FormLabel>
-              <FormControl>
-                <InputField className="flex overflow-hidden pr-0">
-                  <Input placeholder="Untitled" {...field} disabled={isReadOnly} />
-                  <CopyButton
-                    content={field.value}
-                    className="rounded-md rounded-s-none border-b-0 border-r-0 border-t-0 text-neutral-400"
+    <>
+      <PageMeta title={workflowName} />
```
Comment (LetItRock) [inline apps/dashboard/src/components/workflow-editor/configure-workflow.tsx:30]: page meta component

**Review by SokratisVidros — APPROVED** [review]

**Pull request merged**